src/custom/components/Header/Polling/index.tsx

```diff
@@ -1,28 +1,67 @@
 import styled from 'styled-components/macro'
-import PollingUni, { StyledPolling, StyledPollingDot, Spinner } from './PollingMod'
+import PollingUni, { StyledPolling, StyledPollingDot, StyledPollingNumber, StyledGasDot, Spinner } from './PollingMod'
 
 export * from './PollingMod'
 
 const Wrapper = styled.div`
   ${StyledPolling} {
-    color: ${({ theme }) => theme.footerColor};
-    opacity: 0.3;
+    color: ${({ theme }) => theme.text1};
+    position: relative;
+    margin: 0;
+    padding: 0;
+    right: initial;
+    bottom: initial;
+    font-size: 11px;
+    opacity: 1;
 
-    &:hover {
+    a {
+      color: ${({ theme }) => theme.text1};
+      opacity: 0.5;
+      transition: opacity 0.3s ease-in-out;
+      text-decoration: none;
+
+      &:hover {
+        opacity: 1;
+        text-decoration: underline;
+      }
+    }
+
+    ${StyledPollingNumber}:hover + ${StyledPollingDot} {
+      opacity: 1;
+    }
+
+    ${StyledPollingNumber} > a {
       opacity: 1;
+      color: ${({ theme }) => theme.text1};
+
+      &:hover {
+        opacity: 1;
+        text-decoration: underline;
+      }
     }
   }
 
+  ${StyledGasDot},
+  ${StyledPollingDot} {
+    background: ${({ theme }) => theme.text1};
+  }
+
   ${StyledPollingDot} {
-    background: ${({ theme }) => theme.footerColor};
+    opacity: 0.5;
+  }
+
+  ${StyledGasDot} {
+    width: 4px;
+    height: 4px;
+    margin: 0 0 0 8px;
   }
 
   ${Spinner} {
-    border-left: 2px solid ${({ theme }) => theme.footerColor};
+    border-left: 2px solid ${({ theme }) => theme.text1};
   }
 `
 
-export default function Polling() {
+export const Polling = () => {
```

Comment:
I'm in favour of no export defaults, but why not keep it a function?
You can remove only the `default` keyword if that's what you want here

Reply:
Reverted. I think I saw some TS errors, but turns out those were 'delayed' errors. All good now.